src/v2/BaseLoginRouter.js

```diff
@@ -29,12 +29,19 @@ import uiSchemaTransformer from './ion/uiSchemaTransformer';
 import i18nTransformer from './ion/i18nTransformer';
 import AppState from './models/AppState';
 import idx from 'idx';
+import { OKTA_STATE_TOKEN_KEY } from './view-builder/utils/Constants';
+
+const setStateHandleInStorage = (stateToken) => {
+  if(!sessionStorage.getItem(OKTA_STATE_TOKEN_KEY) && stateToken) {
+    sessionStorage.setItem(OKTA_STATE_TOKEN_KEY, stateToken);
+  }
+};
 
 const startLoginFlow = (settings) => {
+  let stateHandle = sessionStorage.getItem(OKTA_STATE_TOKEN_KEY);
```

Comment:
the stateHandle is being loaded from storage, but the codeVerifier will be re-generated ahead on line 51. Do we need to save the codeVerifier also? If we use a new codeVerifier we will be doing a new PKCE flow. Maybe we want that?

Reply:
interact will only be called when there is no stateToken (either from client or from storage).. So interact call will always be new.